app/app.go

```diff
@@ -481,6 +481,8 @@ func New(
 		app.SlashingKeeper,
 		app.AuthorityKeeper,
 		app.LightclientKeeper,
+		app.BankKeeper,
+		app.AccountKeeper,
```

Comment:
Can you explain why is it needed? Any alternative considered?
We should avoid increasing the complexity of the codebase for non simulation related logic

Reply:
We use a util provided by the cosmos simulation package, which needs these two functions. 
```
// AccountKeeper defines the expected account keeper used for simulations (noalias)
type AccountKeeper interface {
	GetAccount(ctx sdk.Context, addr sdk.AccAddress) types.AccountI
}

// BankKeeper defines the expected interface needed to retrieve account balances.
type BankKeeper interface {
	SpendableCoins(ctx sdk.Context, addr sdk.AccAddress) sdk.Coins
}
```
Both functions are needed to fetch the account to broadcast the simulated msg.


I have not deeply researched alternatives, I did check cosmos modules and modules of some other projects which have simulation tests, and all have the two keepers added. 
Both required functions however are queries that do not modify the state .